src/support/string.cpp

```diff
@@ -152,7 +152,7 @@ std::optional<uint32_t> takeWTF8CodePoint(std::string_view& str) {
 
   uint8_t leading = str[0];
   size_t trailingBytes;
-  uint32_t u;
+  uint32_t u = 0;
```

Comment:
```suggestion
  // This initial value is not needed, and only works around a compiler coverage issue.
  uint32_t u = 0;
```
This seems useful as otherwise the reader might think it's a bug, or that they are misreading the code.